data/mods/innawood/innawood_map_settings.json

```diff
@@ -0,0 +1,544 @@
+      "trail_center_variance": 3,
+      "trail_width_offset_min": 1,
+      "trail_width_offset_max": 3,
+      "clear_trail_terrain": false,
+      "trail_terrain": { "t_dirt": 1 },
+      "trailheads": { "trailhead_basic": 1 }
+    },
+    "map_extras": {
+      "forest": {
+        "chance": 20,
+        "extras": {
+          "mx_portal": 3,
+          "mx_crater": 10,
+          "mx_portal_in": 3,
+          "mx_grass": 20,
+          "mx_grass2": 40,
+          "mx_spider": 200,
+          "mx_nest_wasp": 100,
+          "mx_grove": 500,
+          "mx_shrubbery": 500,
+          "mx_clearcut": 125,
+          "mx_pond": 125,
+          "mx_pond_forest": 125,
+          "mx_pond_forest_2": 125,
+          "mx_clay_deposit": 125,
+          "mx_point_dead_vegetation": 500,
+          "mx_point_burned_ground": 500
+        }
+      },
+      "forest_thick": {
+        "chance": 20,
+        "extras": {
+          "mx_portal": 3,
+          "mx_crater": 10,
+          "mx_portal_in": 3,
+          "mx_grass": 10,
+          "mx_grass2": 20,
+          "mx_shia": 1,
+          "mx_spider": 200,
+          "mx_nest_wasp": 100,
+          "mx_jabberwock": 1,
+          "mx_grove": 500,
+          "mx_shrubbery": 500,
+          "mx_clearcut": 125,
+          "mx_pond": 125,
+          "mx_pond_forest": 125,
+          "mx_pond_forest_2": 125,
+          "mx_clay_deposit": 125,
+          "mx_point_dead_vegetation": 500,
+          "mx_point_burned_ground": 500
+        }
+      },
+      "forest_water": {
+        "chance": 20,
+        "//": "weights normalised to around 1000",
+        "extras": {
+          "mx_portal": 8,
+          "mx_crater": 20,
+          "mx_portal_in": 7,
+          "mx_spider": 200,
+          "mx_grass": 10,
+          "mx_grass2": 20,
+          "mx_pond": 240,
+          "mx_pond_swamp": 240,
+          "mx_pond_swamp_2": 240,
+          "mx_clay_deposit": 140,
+          "mx_point_dead_vegetation": 60,
+          "mx_nest_dermatik": 50
+        }
+      },
+      "field": {
+        "chance": 6,
+        "extras": {
+          "mx_portal": 1,
+          "mx_crater": 15,
+          "mx_portal_in": 1,
+          "mx_point_dead_vegetation": 50,
+          "mx_grass": 350,
+          "mx_grass2": 350,
+          "mx_trees": 50,
+          "mx_trees2": 80,
+          "mx_Trapdoor_spider_den": 4,
+          "mx_pond": 20,
+          "mx_point_burned_ground": 50,
+          "mx_nest_wasp": 5
+        }
+      },
+      "road": { "chance": 750, "extras": { "mx_bandits_block": 80, "mx_portal": 5, "mx_crater": 10, "mx_portal_in": 4 } },
+      "marloss": { "chance": 20, "extras": { "mx_marloss_pilgrimage": 100 } },
+      "river": { "chance": 3, "extras": { "mx_reed": 100 } },
+      "lake_shore": { "chance": 2, "extras": { "mx_reed": 100 } }
+    },
+    "city": {
+      "shop_radius": 0,
+      "shop_sigma": 0,
+      "park_radius": 0,
+      "park_sigma": 0,
+      "houses": { "house_w_1": 50 },
```

Comment:
This house still spawns is intended or just an oversight?

Reply:
Leaving these lists empty caused errors. I managed to get rid of the houses by making roads unable to be placed instead.